shopify/resources/variant.py

```diff
@@ -8,12 +8,19 @@ class Variant(ShopifyResource, mixins.Metafields):
     @classmethod
     def _prefix(cls, options={}):
         product_id = options.get("product_id")
-        if product_id:
+        if options.get('is_new'):
```

Comment:
IIRC, we were getting the incorrect URI for Options. This specific change is OBE for us anyway as our solution won't be using Options anymore anyway.